mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,15 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.deleteOrder = async (req, res) => {
+  try {
+    const order = await Order.findByIdAndRemove(req.params.orderId);
```

Comment:
The variable `order` is not checked for null or undefined before being used in the subsequent code. It should be checked to avoid potential null pointer exceptions.